assets/pages/new-contributions-landing/components/ContributionSubmit.jsx

```diff
@@ -43,8 +45,8 @@ function ContributionSubmit(props: PropTypes) {
           spokenCurrencies[props.currency],
           props.amount,
           false,
-        ) : null}
-        {getFrequency(props.contributionType)}&nbsp;
+        ) : null}&nbsp;
+        {freq ? `${freq} ` : null}
```

Comment:
we don't usually use abbreviations in this codebase, so `frequency` would be better